exporter/dorisexporter/exporter_common.go

```diff
@@ -0,0 +1,109 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package dorisexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dorisexporter"
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"fmt"
+	"net/http"
+	"time"
+
+	_ "github.com/go-sql-driver/mysql" // for register database driver
+	"go.uber.org/zap"
+)
+
+const timeFormat = "2006-01-02 15:04:05.999999"
+
+type commonExporter struct {
+	client *http.Client
+
+	logger   *zap.Logger
+	cfg      *Config
+	timeZone *time.Location
+}
+
+func newExporter(logger *zap.Logger, cfg *Config) (*commonExporter, error) {
+	client := &http.Client{
```

Comment:
maybe look at https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/confighttp#client-configuration to create a HTTP client that is going to have the ability to take on extensions and be configured for security.